.github/workflows/branch.yml

```diff
@@ -0,0 +1,44 @@
+name: nf-core branch protection
+# This workflow is triggered on PRs to master branch on the repository
+# It fails when someone tries to make a PR against the nf-core `master` branch instead of `dev`
+on:
+  pull_request_target:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      # PRs to the nf-core repo master branch are only ok if coming from the nf-core repo `dev` or any `patch` branches
+      - name: Check PRs
+        if: github.repository == 'nf-core/nascent'
+        run: |
+          { [[ ${{github.event.pull_request.head.repo.full_name }} == nf-core/nascent ]] && [[ $GITHUB_HEAD_REF = "dev" ]]; } || [[ $GITHUB_HEAD_REF == "patch" ]]
```

Comment:
I think there was something discussed on slack whether double quotes may be needed again, commenting so I won't forget to look this up....

Reply:
hmmmm I don't see the branch protection test fail or pass. did you turn it of?

Reply:
Not that I know of, do you know how to check?